.travis.yml

```diff
@@ -43,11 +41,12 @@ install:
   - if [[ "$DB" = "postgresql" ]]; then pip install psycopg2; fi
 
   # Setup application
-  - if [[ "$DB" = "sqlite" ]]; then invoke create_settings; fi
-  - if [[ "$DB" = "postgresql" ]]; then invoke create_settings --database-type postgresql; fi
+  - if [[ "$DB" = "sqlite" ]]; then invoke create-settings; fi
+  - if [[ "$DB" = "postgresql" ]]; then invoke create-settings --database-type postgresql; fi
 
 # Create test databases
 before_script:
+  - npm install --global gulp-cli
```

Comment:
Great work on fixing the gulp linting, the build is working as it should successfully.

Reply:
Thanks @Rhytah